python/cudnn/gemm/cutedsl/dense/amax/jax_api.py

```
@@ -0,0 +1,144 @@
+    Arguments are JAX arrays (or tracers): A (M, K, 1) and B (N, K, 1) k-major
+    C-contiguous, SFA/SFB in the physical atom shape (1, MN', K', 32, 4, 4).
+    Returns a plain ``(c_tensor, amax_tensor)`` tuple of fresh JAX arrays (a plain
+    tuple, not a TupleDict, so the result is a valid JAX pytree under jit); C is n-major.
+
+    Note: calling this eagerly re-traces the ffi_call each time -- prefer calling it
+    from inside a jitted function in hot loops.
+    """
+    c_dtype = _convert_to_cutlass_data_type(c_dtype)
+    acc_dtype = _convert_to_cutlass_data_type(acc_dtype)
+
+    m, _, l = a_tensor.shape
```

Ruff (0.16.1)

[error] Ambiguous variable name: `l`

(E741)